src/assimp/assimp.cpp

```diff
@@ -822,9 +841,13 @@ vsg::ref_ptr<vsg::Node> SceneConverter::visit(const aiNode* node, int depth)
         }
     }
 
-    if (children.empty()) return {};
+    if (children.empty())
```

Comment:
I think a simpler implementation would be to have:

if (children.empty() && !honour_empty_nodes) return {};

This would also avoid the need extra help notToTransform() function.